server/src/collection.rs

```diff
@@ -1,15 +1,34 @@
 // org.freedesktop.Secret.Collection
 
-use std::collections::HashMap;
+use std::{
+    collections::HashMap,
+    sync::{atomic::AtomicBool, Arc},
+    time::{Duration, SystemTime},
+};
 
 use oo7::dbus::api::{Properties, SecretInner};
+use tokio::sync::{Mutex, RwLock};
 use zbus::{interface, zvariant};
 use zvariant::{ObjectPath, OwnedObjectPath};
 
 use super::Result;
+use crate::{item, service_manager::ServiceManager};
 
 #[derive(Debug)]
-pub struct Collection {}
+#[allow(unused)] // todo: remove this after completing the ifce impl
+pub struct Collection {
+    // Secret Collection properties
```

Comment:
```suggestion
    // Properties
```

Reply:
None of those are exposed as properties though... I suppose would be done later?

Reply:
yes :)